### PR TITLE
Update secure section of the tutorial with new sidecars generation feature

### PR DESCRIPTION
**Pull request by leandroberetta**
This PR adds new comments and steps to explain the new sidecars generated by "Create traffic policies".

Feature PR: [#2247](https://github.com/kiali/kiali-ui/pull/2247)

### CODE REVIEW

**Comment by leandroberetta** [comment]
To see the new additions:

https://deploy-preview-420--kiali.netlify.app/documentation/staging/tutorial/#01-traffic-policies

File: content/documentation/staging/tutorial/06-secure/01-traffic-policies.adoc [hunk 1]
```diff
@@ -14,7 +14,9 @@ In this tutorial we will show how Kiali can use telemetry information to create
 
 Istio telemetry aggregates the ServiceAccount information used in the workloads communication. This information can be used to define authorization policies that deny and allow actions on future live traffic communication status.
 
-This step will show how we can define authorization policies for the *travel-agency* namespace in the Travel Demo application, for all existing traffic in a given time period.
+Besides authorization policies, sidecars can be created to limit the hosts a given workload can communicate with. This helps to better control traffic and also to reduce the https://istio.io/latest/docs/ops/deployment/performance-and-scalability/#cpu-and-memory[memory footprint, window="_blank"] of the proxies.
```
Comment (lucasponce) [inline content/documentation/staging/tutorial/06-secure/01-traffic-policies.adoc:17]: This is style and probably @jshaughn may have better suggestions than me on this.

But I'd start the phrase like:

"Additionally, Istio sidecars can be created to ..."

File: content/documentation/staging/tutorial/06-secure/01-traffic-policies.adoc [hunk 2]
```diff
@@ -37,13 +39,17 @@ We should validate that telemetry has updated the *travel-portal* namespace and
...
 
-NOTE: *Step 2:* Create Authorization Policies for current traffic for *travel-agency* namespace
+NOTE: *Step 2:* Create Authorization Policies and Sidecars for current traffic for *travel-agency* namespace
 
 Every workload in the cluster uses a https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/[Service Account, window="_blank"].
 
 *travels.uk*, *viaggi.it* and *voyages.fr* workloads use the default *cluster.local/ns/travel-portal/sa/default* ServiceAccount defined automatically per namespace.
 
-This information is propagated into the Istio Telemetry and Kiali can use it to define a set of AuthorizationPolicy rules using the "Create Traffic Policies" action located in the Overview page.
+This information is propagated into the Istio Telemetry and Kiali can use it to define a set of AuthorizationPolicy rules.
```
Comment (lucasponce) [inline content/documentation/staging/tutorial/06-secure/01-traffic-policies.adoc:48]: --> "a set of AuthorizationPolicy rules and Sidecars" 

And below I'd add

"Sidecars restrict the list of hosts each workload can communicate with using the current traffic" ?

File: content/documentation/staging/tutorial/06-secure/01-traffic-policies.adoc [hunk 3]
```diff
@@ -126,6 +149,19 @@ Using "Outbound Metrics" tab from the *loadtester* workload we can group per "Re
 </a>
 ++++
 
-NOTE: *Step 5:* Update or delete Istio Configuration
+NOTE: *Step 5:* Verify the proxies clusters list is limited by the Sidecars
+
+According to https://istio.io/latest/docs/reference/config/networking/sidecar/[Sidecar, window="_blank"] documentation, Istio will program all sidecar proxies in the mesh with the necessary configuration required to reach every workload instance in the mesh. After the sidecars are created, the lists of hosts is reduced according to the current traffic. To verify this, we can look for the clusters configured in each proxy.
```
Comment (lucasponce) [inline content/documentation/staging/tutorial/06-secure/01-traffic-policies.adoc:154]: Not sure if it's me but "Istio will program" -> "Istio will configure" sounds perhaps more straightforward/clear ?

**Review by jshaughn — CHANGES_REQUESTED** [review]
Nothing wrong with the PR but I think actually I want to freeze ay changes to master.  I will add myself as an assignee here, and will work the changes into the docsy branch, which is already updated with other content and will replace master befoer the next release.

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
We'll want to resubmit this PR once the new docsy website is in place. Putting a block on this PR so it doesn't get merged.

**Comment by israel-hdez** [comment]
@leandroberetta This was automatically closed when the `master` branch was deleted.
Please, prepare a new PR. The `master` branch no longer exists. You should now target the `staging` branch.

**Comment by jshaughn** [comment]
@leandroberetta no, don't worry about a new PR.  I will handle it, thanks!

**Pull request closed without merging**